tests/run/kwargs_passthrough.pyx

```diff
@@ -1,22 +1,16 @@
 cimport cython
 
 
-@cython.test_fail_if_path_exists('//MergedDictNode')
-def wrap_passthrough(f):
+def modify_in_function(**kwargs):
     """
-    >>> def f(a=1): return a
-    >>> wrapped = wrap_passthrough(f)
-    >>> wrapped(1)
-    CALLED
-    1
-    >>> wrapped(a=2)
-    CALLED
-    2
+    >>> d = {'foo': 'bar'}
+    >>> modify_in_function(**d)
```

Comment:
This doesn't do what you think. The code in the docstring is run in Python. The real test is something like:

```
def modify_in_function():
    def inner(**kwds):
        kwds['foo'] = 'modified'
    d = {'foo': 'bar'}
    print(d)
    inner(**d)
    print(d)
```

That way `inner(**d)` is in the bit that's compiled in Cython.